ansible/configs/ocp-storage-cns/post_ocp_nfs_config.yml

```diff
@@ -26,15 +27,15 @@
       template:
         src: "./files/pvs.j2"
         dest: "/root/pvs-{{ env_type }}-{{ guid }}.yml"
-      tags: [ gen_pv_file ]
+      tags: [gen_pv_file]
```

Comment:
I think this would be more readable if it were like

```
tags:
  - gen_pv_file
```

Plus you use the other list format later in this file, so consistency!

Reply:
fixed